docs/source/blocks/editcomponent.md

```diff
@@ -127,6 +127,43 @@ import { Icon } from '@plone/volto/components';
 </SidebarPortal>;
 ```
 
+### Default values support for block schema forms
+
+In the block schema definition, you can use the two additional properties for
+each field: `default` and `missing`. The `default` value is used to populate
+the field with an initial value. The `missing` value should be used when the field
+should always have a value (as required).
+
+```
+  // ...
+  properties: {
+    url: {
+      title: 'Embed URL',
+      default: 'https://example.com',
+      missing: 'https://nothing.com',
+    },
+  }
+  // ...
+```
+
+In effect, the following situation matrix will occur:
+
+- `default` and `missing` are declared: the block form is initialized with
```

Comment:
Nice. I just gave it a try on local vanilla volto. When both the values "default" and "missing" are used, it somehow re-initializes the form field with default value. Here's the schema snippet I created for demo field:
```
demo: {
        title: 'demo',
        widget: 'url',
        default: 'www.default.com',
        missing: 'www.missing.com',
      },

 ```
 
  In addition to this, idk if its a glitch or the way it works, but the values let it be "missing" or "default" started to re-appear after clearing them and editing the form second time🤔

Reply:
@nileshgulia1 that's the effect of setting `missing`. If you don't want that effect, don't set the `missing`.

Btw, this is also the default with current Volto. Try it with this widget:

```
{
        title: 'demo',
        widget: 'url',
        default: 'www.default.com',
      },
```

Reply:
This is how the default behaviour works:

https://user-images.githubusercontent.com/22280901/226318877-30578d6a-51c9-4ba9-b172-4a671d2c9898.mp4